src/old-components/NetworkModal/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { useEffect, useState } from 'react';
 import styled from 'styled-components';
-import { observer } from 'mobx-react';
+// import { observer } from 'mobx-react';

```

Comment:
Should we just delete it?

Reply:
Sure, had left it since I wasnt sure it might break something but it seems fine